src/app/left-menu/main/left-menu.component.ts

```diff
@@ -1,12 +1,11 @@
 import { Component, ElementRef, OnDestroy, OnInit, ViewChild } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
-import { LeftMenuItem } from '../../types/left-menu-item';
-import { MetaItem } from '../../types/meta-item';
-import { LeftMenuAlias } from '../left-menu.aliases';
-
 import { Subscription } from 'rxjs';
+import { LeftMenuItem } from 'src/app/types/left-menu-item';
```

Comment:
why not `from '../../types/left-menu-item'`?

Reply:
vs code auto-import :)